web/themes/custom/server_theme/src/js/expanding-text.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file
+ * JS for button controls.
+ */
+(function ($, Drupal, once) {
+  Drupal.behaviors.expandingText = {
+    attach: function(context, settings) {
+      const $expanding_text = $(once('expanding-text', '.js-expanding-text', context));
```

Comment:
Let's convert to camelCase as per https://www.drupal.org/docs/develop/standards/javascript-coding-standards/javascript-coding-standards#camelcasing

```suggestion
      const $expandingText = $(once('expanding-text', '.js-expanding-text', context));
```